lib/go-tc/deliveryservices.go

```diff
@@ -577,6 +578,36 @@ type DeliveryServiceNullableV11 struct {
 	DeliveryServiceRemovedFieldsV11
 }
 
+type GeoLimitCountriesType []string
```

Comment:
This type should probably have a GoDoc comment. Personally I would also add one to its `UnmarshalJSON` and `MarshalJSON` methods, but those are extremely common especially in our codebase so it probably doesn't add much value.